homeassistant/components/iotawatt/coordinator.py

```diff
@@ -17,6 +17,9 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+# Desired refresh rate (min 5 sec)
+REFRESH_RATE=5
```

Comment:
Don't change the default from the existing. It is an unnecessarily low interval and will put too much load on the iotawatt, effictively reducing its accuracy.

Iotawatt author has stated his concern on similar suggestions many times in the past

Reply:
Is there official documentation from IoTaWatt that says this?  Their own web-UI has a 5 second refresh rate, so I can't imagine their own software would be configured to a rate that creates problems.  I've been running this for many months without issue.

It needs to be something more frequent than the existing 30/60 seconds, that isn't enough to capture useful actions like accurate power draw for a sump-pump cycling or clothes dryer door opening and closing that can result in missed states or automation.

Reply:
But you aren’t running HA website 24/7 nor will you be fetching the value of every single input and output and then make it calculate the energy figure again for all those sensors again (which requires to fetch everything from midnight again. 

You have one need, not every one has that need. If you want something like this then make the option configurable (finish your TODO) so you can configure it for your own need without impacting every one else.